src/arrayMethodSort.js

```diff
@@ -4,8 +4,20 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => (String(a) > String(b) ? 1 : -1),
+  ) {
+    const result = this;
+
+    for (let i = 0; i < this.length - 1; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (compareFunction(this[i], this[j]) > 0) {
+          [this[i], this[j]] = [this[j], this[i]];
+        }
+      }
```

Comment:
The bubble sort algorithm is correctly implemented, but it is not the most efficient sorting algorithm, especially for large datasets. For better performance, consider implementing a more efficient sorting algorithm like quicksort or mergesort in the future.